modules/panorama/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">=0.12.29, <0.14"
+  required_version = ">=0.12.29, <=0.15"
```

Comment:
Using this module, can you `terraform apply` an example successfully on version 0.15.0? I'd be much surprised, which is why I ask.

Maybe let's do that in a separate PR, even if solely for Changelog-related reasons.

Reply:
This is the version I have installed:
```bash
$ terraform version
Terraform v0.14.3
+ provider registry.terraform.io/hashicorp/azurerm v2.56.0
+ provider registry.terraform.io/hashicorp/random v3.1.0

Your version of Terraform is out of date! The latest version
is 0.15.0. You can update by downloading from https://www.terraform.io/downloads.html
```

Terraform apply with TF 0.15.0 below, didn't work prior to the change:
```bash
$ terraform apply -auto-approve

module.panorama.azurerm_managed_disk.this["disk_name_1"]: Refreshing state... [id=/subscriptions/137***016/resourceGroups/salsop_panorama/providers/Microsoft.Compute/disks/pano-deploy-disk-disk_name_1]
module.panorama.azurerm_public_ip.this[0]: Refreshing state... [id=/subscriptions/137***016/resourceGroups/salsop_panorama/providers/Microsoft.Network/publicIPAddresses/ip]
module.panorama.azurerm_network_interface.this: Refreshing state... [id=/subscriptions/137***016/resourceGroups/salsop_panorama/providers/Microsoft.Network/networkInterfaces/mgmt]
module.panorama.azurerm_virtual_machine.panorama: Refreshing state... [id=/subscriptions/137***016/resourceGroups/salsop_panorama/providers/Microsoft.Compute/virtualMachines/pano-deploy]
module.panorama.azurerm_virtual_machine_data_disk_attachment.this["disk_name_1"]: Refreshing state... [id=/subscriptions/137***016/resourceGroups/salsop_panorama/providers/Microsoft.Compute/virtualMachines/pano-deploy/dataDisks/pano-deploy-disk-disk_name_1]

Apply complete! Resources: 0 added, 0 changed, 0 destroyed.
```

Reply:
Oh, a misunderstanding. Please do not try to `apply` a module. That is not how it is supposed to work. Test a module as used in the example:

```
cd example/panorama
terraform init

  Error: ...

```

If you'd like to have 0.15.x, I'd suggest to do it in a separate PR, even if solely for Changelog-related reasons.

Reply:
Yes, you're correct that errors as the TF Example have a version constraint of `required_version = ">=0.13, <0.14"` also, I have not changed this, but I can update this also to match as needed?

Reply:
I was thinking to not to tie 0.15 support to a bugfix regarding the Standard SKU IP. These are not related changes, am I right?

Reply:
Yes, unrelated, I just needed TF 0.15 so added in this. How do I remove this single change? Or can you just approve the other change? Or do I need to do a whole new Pull Request?

Reply:
The simplest way on GitHub is to use the first icon when typing a code review comment. The icon that says `Insert a suggestion <ctrl+g>`. I am doing it right now for you:

```suggestion
  required_version = ">=0.12.29, <0.14"
```

Reply:
If you could `Commit suggestion` now, I'll gladly approve 👍